app/gui2/src/components/GraphEditor.vue

```diff
@@ -1,18 +1,4 @@
 <script lang="ts">
-import { defineKeybinds } from '@/util/shortcuts'
-
-const graphBindings = defineKeybinds('graph-editor', {
-  undo: ['Mod+Z'],
-  redo: ['Mod+Y', 'Mod+Shift+Z'],
-  dragScene: ['PointerAux', 'Mod+PointerMain'],
-  openComponentBrowser: ['Enter'],
-  newNode: ['N'],
-})
-</script>
-
-<script setup lang="ts">
-import { computed, onMounted, reactive, ref, shallowRef, watch } from 'vue'
-
 import { nodeBindings } from '@/bindings/nodeSelection'
 import CodeEditor from '@/components/CodeEditor.vue'
 import ComponentBrowser from '@/components/ComponentBrowser.vue'
```

Comment:
@Frizi just want to confirm that this reordering of imports is intentional?

Reply:
Yes. I found that using separate import lists keeps the IDE auto-format breaking. The only working configuration is if all imports are in the first `script` block in a file. To reduce friction, I think we should simply accept that as a rule.